### PR TITLE
Localize to french

### PR DESCRIPTION
**Pull request by Sheeri**
Localized except for the following (which are either hard to localize, or were added after we gave the translators the phrases:

"Cannot access reports"
"Please, input name or license"
"Issued to"
"Remove" 
"It has been %@ hours after closing the application" 
"It has been %@ hours after starting of your duty."
"Change profile picture" 
"Access your login and password on the keychain"

### CODE REVIEW

File: Localization/es-419.lproj/Localizable.strings [hunk 1]
```diff
@@ -22,7 +22,7 @@
 "No Results for “%@“" = "No Results for “%@“";
 "Ok" = "Ok";
 "Permit #" = "Permit #";
-"Please, input name or licence" = "Please, input name or licence";
+"Please, input name or licence" = "Please, input name or license";
```
Comment (am-MongoDB) [inline Localization/es-419.lproj/Localizable.strings:25]: Better to leave both sides the same until the localisation is done (that way your shell incantation will find that the phrase hasn't been localised yet)

**Review by am-MongoDB — CHANGES_REQUESTED** [review]
We still seem to be missing more phrases
image

**Comment by Sheeri** [comment]
Related to issue #35

File: Localization/es-419.lproj/Localizable.strings (hunk 1 above)
Comment (Sheeri) [reply Localization/es-419.lproj/Localizable.strings:25]: That's just a mispelling in the English Translation, will submit a fix.

As for the missing phrases, I'll add them in, but also look into why "Boardings" isn't translated - it should be "Inspections"......

Comment (Sheeri) [reply Localization/es-419.lproj/Localizable.strings]: submitted a fix for this, now both sides (and the code) say "license".

**Comment by Sheeri** [comment]
I have opened issue #210 to deal with ModalViewButton not using translations, and issue #211 to deal with missing Ukranian translations. I have a few missing translations that I'm waiting for.

**Comment by Sheeri** [comment]
@am-MongoDB This is ready for your review (assuming checks pass, but it built on my machine!)

@mfhan did the actual translating, and it's AWESOME.

**Comment by Sheeri** [comment]
@am-MongoDB localization is complete, can you review? There is issue #210 to deal with ModalViewButton not using translations, and when that's fixed the modal buttons will use the new translations. Other than that I tested everything out.

**Review by am-MongoDB — APPROVED** [review]

**Pull request merged**